Wikipedia/Code/WMFArticleLanguagesSectionFooter.swift

```diff
@@ -0,0 +1,30 @@
+class WMFInterfaceBuilderFriendlyUITableViewHeaderFooterView : UITableViewHeaderFooterView {
+    public override func layoutSubviews() {
+        super.layoutSubviews()
+        // There is no IB UITableViewHeaderFooterView component, which means you have to use a UIView.
+        // But you can't set UITableViewHeaderFooterView's 'contentView' from UIView IB component.
+        // This work-around fixes weird autolayout issues (likely related to this?) where multi-line
+        // labels from a xib using a UITableViewHeaderFooterView subclass appear at the wrong size for
+        // their text length, then jump to thier proper width a second after appearing.
+        frame = CGRect(origin: frame.origin, size: systemLayoutSizeFitting(CGSize(width: bounds.size.width, height: UILayoutFittingCompressedSize.height), withHorizontalFittingPriority: .required, verticalFittingPriority: .defaultHigh)
```

Comment:
A view shouldn't set its own frame inside of `layoutSubviews`, it should only adjust the frame of its subviews.

Reply:
Ya I wasn’t liking that either. Forgot to add Changes Welcome tag :)